besca/pl/_dot_heatmap.py

```diff
@@ -507,28 +507,6 @@ def dot_heatmap_split(adata,
         A matplotlib figure element containing the generated plot. To save the figure this plot will need
         to be passed to a parameter and saved in a second step through the fig.savefig() function call.
 
-    Examples
```

Comment:
Why do we not have any examples here anymore? Do they currently throw errors?

Reply:
Example was on the PBMC storage dataset. which is not releasable...The one I tried out throws an error indeed (on Kotliarev data). It is related to the issue #74. We need to fix this, and find an example that runs. Additionaly 
many datasets did not have import function yet. We can wait to fix it indeed, or try to still have some documentation up to date as we need this in the master to build the html doc.

Reply:
Ah ok. Then remove the example currently but pleaser replace with a placeholder (i.e. a comment explaining this) so we remember to fix this :)

Reply:
This is fix now in 
5ddcd6e